### ISSUE TITLE
Bound-withs are bound to be an issue

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_traject#42]
In GitLab by @mxk128 on Sep 28, 2018, 14:42

How to we index/display Bound-with records. In Symphony these bound-with records don't contain itemids. Some catkeys examples:

```
1006921  
1021944
10018394
```

I will provide more examples.

**Comment by banukutlu** [comment]
In GitLab by @cdm32 on Sep 28, 2018, 14:54

Great issue title @mxk128  I see what you did there

**Comment by ruthtillman** [comment]
Here are some samples of how others have done it: 

Princeton - https://catalog.princeton.edu/catalog/2893853 & https://catalog.princeton.edu/catalog/3023901

Cornell - https://newcatalog.library.cornell.edu/catalog/656258 & https://newcatalog.library.cornell.edu/catalog/1793827

**Comment by ruthtillman** [comment]
To-dos from 10/25 meeting:

* Jeff and Maryam will experiment with different methods and ordering for the exports. They'll see whether we can get at least the call number/library (not location/item) data for the bound-with records. Items would be gravy but seem unlikely based on how Sirsi displays it.
* Charlie is going to ask NC State about how they extract bound-withs.

We may also contact Stanford, but their procedures are older.

**Comment by cdmo** [comment]
I emailed a colleague at NCSU just now, will keep you all posted

**Comment by mkutch** [comment]
Jeff says bound-with notes in Sirsi MARC can appear in one of 3 fields.

- 501
- 590
- 591

They are inconsistently used in the CAT and will not exist in many records for filmed-with microform.

There is some question in our minds (Jeff and Maryam) about how Blacklight displays bound- or filmed- with relationships. The Cornell example given above seems to draw the data from somewhere other than the MARC data, On the other hand, the Princeton example seems to be displaying a 5XX note. Princeton seems to make 5XX fields that are not usually local into local fields by appending a subfield 5.

As we build the 949s based on the code Dawn provided, should we also consider building 5XX fields of some kind for the purposes of display?

**Comment by mkutch** [comment]
Examples in the CAT of bound with records

Parent
https://cat.libraries.psu.edu/uhtbin/cgisirsi/0/0/0/5?searchdata1=^c355035
Some of its Children
https://cat.libraries.psu.edu/uhtbin/cgisirsi/0/0/0/5?searchdata1=^c1591 
https://cat.libraries.psu.edu/uhtbin/cgisirsi/0/0/0/5?searchdata1=^c355055
https://cat.libraries.psu.edu/uhtbin/cgisirsi/0/0/0/5?searchdata1=^c355093

**Comment by banukutlu** [comment]
@cdmo this sounds great to me. I agree that we do not need to index the field because`all_text_timv` 👍

**Comment by banukutlu** [comment]
@mkutch could you pls update https://github.com/psu-libraries/psulib_traject/wiki/Full-and-Incremental-Extracts-from-Symphony for bound with extract?

**Comment by mkutch** [comment]
I noticed that the display of the “Bound in” tag at the title level is misleading. I spoke with Jeff Edmunds about this and he also said the Bound with relationships should never be stated at the tithle level.  See catkey 485144. This title has 289 items under it but only one item is bound with another title(catkey 11749683). 
In the CAT you see that the Bound in label is displayed at the holdings/item level. 
Q - Thoughts

**Comment by ruthtillman** [comment]
@mkutch is it possible to add to the extract exactly which edition is bound in another? Perhaps we can review the field and see what's possible? We can also explore this more when displaying the holdings/availability, but I would prefer to see if we can start by being specific at the top level about what's bound in what and then maybe go from there.

**Comment by ruthtillman** [comment]
Looking at the data in the 949, could we include in the auto-generated 541s something like

949a 949c (949t) bound in [title], e.g.

AY67.N5W7 1922-24 c. 1 (MICROFORM) bound in The World almanac and encyclopedia

or

Microfilm D285 reel.478 no.3575 c.1 (MICROFORM) bound in The high-caste Hindu woman / With introduction by Rachel L. Bodley

I don't know if that means we'd want to separate out a for the title of bound in (and link that) and put the rest in another field? And depending on item types would it make sense to use a map for format?

**Comment by banukutlu** [comment]
@ruthtillman 
yes, we can include `949a 949c (949t)` in one subfield in our  generated`591s` like
```591 a|Parent format, copy number, call number b| Parent title c| Parent catkey```
 or 
if you think we should use a translation map for format, then we will copy `949t` to a separate subfield in our 591s and `949a 949c` in another subfield like 
```591 a|Parent format b| Parent copy number, call number d| Parent title c| Parent catkey```

So, what do you think?

**Comment by ruthtillman** [comment]
For the sake of some symmetry, let's put Parent title in |a, but otherwise I think let's go with 2 if it's feasible.

**Comment by banukutlu** [comment]
ok what about `591 a|Parent title c| Parent catkey t| Parent format n| Parent copy number, call number` for the subfields?

**Comment by mkutch** [comment]
I noticed another issue. See  `https://blackcat01qa.libraries.psu.edu/catalog/988373`. This has a bound with item but isn't showing up. I noticed the librarian view doesn't list all the 949s nor the 591.  I went back to the extract and saw this record was broken down into 8 marc records (by sirsi during catalogdump). Since the 591 build happens after the catalogdump it is on the 8th marc record.  Checked with the extract and it looks like the librarian view is showing only the 7th marc record. 
I can show you what I mean if this sounds confusing.
